source/includes/_oy-tutorial.md

```diff
@@ -187,3 +187,74 @@ You can enable or disable notification settings for disbursement products if you
 2. The email field:  allows the user to add, remove, and edit email notification receivers in this column, which can hold up to three email addresses.
 3. Click ‘Save Changes’
 
+## Multi Account Management
+
+Multi Account Management (Subsidiaries) is a feature that can help you handle complex relations between entities in your business in the Oy! Dashboard. Please contact our Sales or Account Management if you’re interested to use this feature.  
+With this feature you will be able to :
+
+1. Have a detailed report on transactions performed by all of subsidiary account
+2. Distribute balance between the parent account to subsidiaries account
+3. Use subsidiary balance to disburse funds
```

Comment:
tambahin ini mungkin?
`4. free admin fee to disburse between related user`

Reply:
ok gw tambahin..udh confirmed sama mas simon